frontend/src/css/style.css

```diff
@@ -671,3 +671,13 @@ button:disabled {
     }
 
 }
+
+.game-mode-btn-precedent {
```

Comment:
Classname is not right. This component is used in many pages and not only on choose game mode page, call it "back-arrow" or "back-arrow-button"